frontend/appflowy_flutter/lib/plugins/database/widgets/cell/card_cell_skeleton/text_card_cell.dart

```diff
@@ -141,9 +141,8 @@ class _TextCellState extends State<TextCardCell> {
 
   Widget? _buildIcon(TextCellState state) {
     if (state.emoji?.value.isNotEmpty ?? false) {
-      return Text(
+      return FlowyText(
```

Comment:
Using `FlowyText.emoji` is better if `state.emoji?.value` is an emoji.

Reply:
Thanks!